spec/features/user/user_show_spen.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
```

Comment:
- [ ] You should have a test case that asserts you can see the user's profile picture.
- [ ] You should have a test case that asserts you can see the user's first 3 posts.
- [ ] You should have a test case that checks if the app redirects to the post's show page when you click on a user's post.

Reply:
Done ✔